### ISSUE TITLE
--contain flag interferes with --pwd

### ISSUE THREAD

**Issue by kyleam** [apptainer/singularity#1208]
## Version of Singularity:

2.4.1-dist

### Expected behavior

`--pwd` works with `--contain`.

### Actual behavior

`--pwd` no longer has an effect when `--contain` is given.

### Steps to reproduce behavior

```
$ singularity pull shub://vsoch/hello-world
$ singularity exec --pwd /opt vsoch-hello-world-master-latest.simg pwd
$ singularity exec --contain --pwd /opt vsoch-hello-world-master-latest.simg pwd
```

For the first `exec` line, I see "/opt" as I expect.  For the second
one, I see "/home/kyle".

**Comment by kyleam** [comment]
[+cc @yarikoptic]

**Comment by kyleam** [comment]
Closing, since this was fixed in 2.4.3.  Thanks!
